server/tests/helpers/utils.js

```diff
@@ -36,59 +31,12 @@ export const verifyUser = async (agent, data) => {
   return loginUser(agent, data);
 };
 
-export const getCategoryId = async (values) => {
-  try {
-    const [category] = await Category.findOrCreate({
-      where: { name: values },
-      defaults: {
-        name: values
-      },
-    });
-    const { id } = category;
-    return id;
-  } catch (error) {
-    console.log(error);
-  }
-};
-
-export const getUserId = async (email, username) => {
-  try {
-    const user = await User.create(getUserData({
-      email, username
-    }));
-    const { id } = user;
-    return id;
-  } catch (error) {
-    console.log(error);
-  }
-};
-
-export const getTagId = async (values) => {
-  try {
-    const [tag] = await Tag.findOrCreate({
-      where: { name: values },
-      defaults: {
-        name: values
-      },
-    });
-    const { id } = tag;
-    return id;
-  } catch (error) {
-    console.log(error);
-  }
-};
-
-
-export const getArticleId = async (title, body, userId) => {
+export const getModelObjectId = async (model, obj) => {
```

Comment:
Great job done on the code refactoring

Reply:
Thanks. The returns were intended for code readability.